project-loader/src/main/kotlin/Args.kt

```diff
@@ -1,10 +1,19 @@
 package de.itemis.mps.gradle.project.loader
 
-import LogLevel
 import com.xenomachina.argparser.ArgParser
 import com.xenomachina.argparser.default
 import java.io.File
 
+
+class GenerateArgs(parser: ArgParser) : Args(parser) {
```

Comment:
This class doesn't belong here, it's generator-specific.

Reply:
HeikoBecker recommended to move GenerateArgs class into the Args.kt file as it is small enough that it does not need a separate file.